src/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_plugin_utils.py

```diff
@@ -199,7 +199,7 @@ def exception_break(self, py_db, frame, thread, arg, is_unwind=False):
 
         return None
 
-    def change_variable(self, frame, attr, expression):
+    def change_variable(self, frame, attr, expression, scope=None):
         for plugin in self.active_plugins:
             ret = plugin.change_variable(frame, attr, expression, self.EMPTY_SENTINEL)
```

Comment:
Should the `scope` be passed along to the plugins within the loop?

Reply:
Yes I should probably do that. They don't use it, but makes sense